pkg/option/config.go

```diff
@@ -2106,10 +2120,8 @@ func (c *DaemonConfig) Populate() {
 		c.LogOpt = m
 	}
 
-	if val := viper.GetInt(ConntrackGarbageCollectorIntervalDeprecated); val != 0 {
-		c.ConntrackGCInterval = time.Duration(val) * time.Second
-	} else {
-		c.ConntrackGCInterval = viper.GetDuration(ConntrackGCInterval)
```

Comment:
:thinking: how did we end up dropping the conntrackGCInterval?

Reply:
Thanks. Fixed